R/GitHost.R

```diff
@@ -63,20 +65,26 @@ GitHost <- R6::R6Class("GitHost",
     },
 
     # Get repositories URLS from the Git host
-    get_repos_urls = function(type = "web", code = NULL, file = NULL, verbose, settings) {
+    get_repos_urls = function(type = "web",
```

Comment:
❯ checking Rd \usage sections ... WARNING
  Undocumented arguments in documentation object 'get_repos_urls'
    ‘in_files’

Reply:
Should we add `cache` parameter for consistency with other functions?

Reply:
```
> in_with_files <- get_repos_urls(git_stats, in_files = "DESCRIPTION", with_files = "renv.lock")
ℹ [Host:GitLab][Engine:REST][Scope:RWDInsightsEngineering] Pulling repositories...
ℹ Searching for code [renv.lock]...
ℹ [Host:GitHub][Engine:REST][Scope:RWDScodeshare] Pulling repositories...       
ℹ Searching for code [renv.lock]...
```

Since `in_files` parameter should work only with `with_code`, it would be good to add proper warning. And from the verbose message we can see that it checks for code `renv.lock`